data/EndGaurav.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "Gaurav Singh Jethuri",
+  "displayStatsPublic": true,
```

Comment:
This `displayStatsPublic` key is no longer needed as of our latest update, so please update it accordingly. 
```suggestion
```